huxley/www/js/components/RegistrationView.js

```diff
@@ -724,4 +614,484 @@ const RegistrationPhoneInput = React.createClass({
   },
 });
 
-module.exports = RegistrationView;
+const RegistrationHeader = React.createClass({
```

Comment:
This is actually a question that I have. Should we put each of these components into their own file even if we'll never use them outside of the RegistrationView?